src/skin/qml/qmllibrarytracklistmodel.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <QIdentityProxyModel>
+
+#include "library/librarytablemodel.h"
+
+namespace mixxx {
+namespace skin {
+namespace qml {
+
+class QmlLibraryTrackListModel : public QIdentityProxyModel {
+  public:
+    enum Roles {
+        TitleRole = Qt::UserRole,
+        ArtistRole,
+        AlbumRole,
+        AlbumArtistRole,
+        FileUrlRole,
+    };
+    Q_ENUM(Roles);
+
+    QmlLibraryTrackListModel(LibraryTableModel* pModel, QObject* pParent = nullptr)
```

Comment:
librarytablemodel.h does not need to be included when moving this implementation into the .cpp file.

Reply:
07b2ec088973dbac277851559f7eb1c1300e53f8